<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QMainWindow" name="SignupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background: rgb(234, 230, 224);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Logo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Logo.qrc">:/image/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
color: rgb(28, 28, 28);
border: 1.5px solid #535366;
border-radius: 5px; </string>
     </property>
     <property name="placeholderText">
      <string>이름을입력하세요.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswdInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF;
color: rgb(28, 28, 28);
border: 1.5px solid #535366;
border-radius: 5px; </string>
     </property>
     <property name="placeholderText">
      <string>비밀번호를 입력하세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; 
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }</string>
     </property>
     <property name="text">
      <string>가입하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
    background-color: #535366; 
    color: white; 
    border-radius: 5px; 
    }
 QPushButton:hover { 
    background-color: #555; 
    }
  QPushButton:pressed { 
    background-color: #777; 
    }</string>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
